plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -813,11 +830,11 @@ static void encode_audio(struct ffmpeg_output *output,
 	if (!got_packet)
 		return;
 
-	packet.pts = rescale_ts(packet.pts, context, data->audio->time_base);
-	packet.dts = rescale_ts(packet.dts, context, data->audio->time_base);
+	packet.pts = rescale_ts(packet.pts, context, data->audio_streams[idx]->time_base);
```

Comment:
Style:  This line (and the one below) are now a bit too long.  Please line break them.